### PR TITLE
🔀 Fix/redesign vehicle parser

### PR DESCRIPTION
**Pull request by jxn-30**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [x] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**What is included in your update?**
* this PR will contain the bugfix for #1990 

**Additional notes**
–

### CODE REVIEW

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Comment by deoxis9001** [comment]
```
    "patientIntermediate": {
        "caption": "Station de Sauvetage",
        "dispatch": "Transférer les patients",
        "home": "Bât : {building}"
    },
    "prisonerIntermediate": {
        "caption": "Station de Sauvetage",
        "dispatch": "Transférer les prisonniers",
        "home": "Bât : {building}"
    },
```
```
        "patientIntermediate": {
            "distance": "Distance maximum",
            "home": "Bâtiments",
            "show": "Montrer x bâtiments de chaque",
        },
        "prisonerIntermediate": {
            "distance": "Distance maximum",
            "home": "Bâtiments",
            "show": "Montrer x bâtiments de chaque",
        },
 ```
fr_FR

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Comment by jxn-30** [comment]
> ```
>     "patientIntermediate": {
>         "caption": "Station de Sauvetage",
>         "dispatch": "Transférer les patients",
>         "home": "Bât : {building}"
>     },
>     "prisonerIntermediate": {
>         "caption": "Station de Sauvetage",
>         "dispatch": "Transférer les prisonniers",
>         "home": "Bât : {building}"
>     },
> ```
> 
> ```
>         "patientIntermediate": {
>             "distance": "Distance maximum",
>             "home": "Bâtiments",
>             "show": "Montrer x bâtiments de chaque",
>         },
>         "prisonerIntermediate": {
>             "distance": "Distance maximum",
>             "home": "Bâtiments",
>             "show": "Montrer x bâtiments de chaque",
>         },
> ```
> 
> fr_FR

There are two new ones that I forgot on previous commits @deoxis9001 :
https://github.com/LSS-Manager/LSSM-V.4/blob/f9718803daaa27a57b0aaacf0895b46abe876f12/src/modules/redesign/i18n/fr_FR/vehicle.json#L128
https://github.com/LSS-Manager/LSSM-V.4/blob/f9718803daaa27a57b0aaacf0895b46abe876f12/src/modules/redesign/i18n/fr_FR/vehicle.json#L130

These are the tab titles:
grafik

The whole window for `patient-intermediate` and `prisoner-intermediate` looks like this:
grafik

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Comment by deoxis9001** [comment]
``` "patientIntermediate": "Station de Sauvetage", ```
``` "prisonerIntermediate": "Station de Sauvetage", ```
fr_FR

File: src/modules/redesign/parsers/vehicle.ts [hunk 1]
```diff
@@ -318,112 +287,355 @@ export default <RedesignParser<VehicleWindow>>(({
...
+                        department: !!row.children[isOwn ? 3 : 4].querySelector(
+                            '.label.label-success'
+                        ),
+                        list,
+                        tax: isOwn
+                            ? 0
+                            : parseInt(
+                                  row.children[3]?.textContent?.trim() ?? '-1'
+                              ),
+                        state: alarmEl?.classList.contains('btn-success')
```
Comment (Cr4zyc4k3) [inline src/modules/redesign/parsers/vehicle.ts:437]: Little bit ninja coding ;)

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**